tools/ci_build/build.py

```diff
@@ -487,8 +487,10 @@ def run_onnx_tests(build_dir, configs, onnx_test_data_dir, provider, enable_para
         cmd = []
         if provider:
           cmd += ["-e", provider]
-          if provider == 'cuda':
-            cmd += ["-j", "2"]
+
+        if num_parallel_models > 0:
+          cmd += ["-j", str(num_parallel_models)]
```

Comment:
So, we won't test the builtin cpu provider with num_parallel_models>0?

Reply:
plz check the parameters set on line 600 onwards, I believe, the execution flow remains the same for the 3 cases that sets num_parallel_models = 1